opencti-platform/opencti-front/src/private/components/entities/organizations/OrganizationEditionOverview.js

```diff
@@ -1,22 +1,11 @@
-import React, { Component } from 'react';
-import * as PropTypes from 'prop-types';
-import { graphql, createFragmentContainer } from 'react-relay';
-import { Formik, Form, Field } from 'formik';
-import withStyles from '@mui/styles/withStyles';
-import {
-  assoc,
-  compose,
-  map,
-  pathOr,
-  pipe,
-  pick,
-  difference,
-  head,
-} from 'ramda';
+import React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+import { Field, Form, Formik } from 'formik';
+import * as R from 'ramda';
+import { assoc, difference, head, map, pathOr, pick, pipe } from 'ramda';
```

Comment:
It is better to import Ramda as you did line 4, then call the different method directly on it : R.assoc, R.head etc. Ramda becomes easier to search this way.

Reply:
Very good point